docs/user-guide/install.md

```diff
@@ -91,3 +91,60 @@ vsphere
 ID                  NAME                DESCRIPTION         ENABLED
 ```
 
+## Installation on Windows Docker Hosts
```

Comment:
we should not have the same text in 2 places. Readme.md can refer to this file. Please do not copy-n-paste chunks of docs